src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -350,15 +350,15 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
         m_image_id{0} {
             if (m_vlm_config.model_type == VLMModelType::MINICPM) {
                 m_resampler = ov::Core{}.compile_model(
-                    model_dir / "resampler.xml", device, device_config
+                    model_dir / "openvino_resampler_model.xml", device, device_config
                 ).create_infer_request();
 
                 m_embedding = ov::Core{}.compile_model(
-                    model_dir / "embed_tokens.xml", device, device_config
+                    model_dir / "openvino_text_embeddings_model.xml", device, device_config
                 ).create_infer_request();
 
                 m_language = ov::Core{}.compile_model(
-                    model_dir / "language_model.xml", device, device_config
+                    model_dir / "openvino_language_model.xml", device, device_config
                 ).create_infer_request();
```

Comment:
With optimum-cli export some models (`m_embedding` and `m_language`) will have the same names for minicpm and llava use cases. It can be moved to a common part above, so we will not need if-statement branch for llava and llava next (only for minicpm with its resampler model). Probably it can be applied in further PRs.

Reply:
Moved